app/propsGenerators/indexPagePropsGenerator.ts

```diff
@@ -25,6 +31,33 @@ const getCachedLatestFetchyRun = createCache(getLatestFetchyRun, {
   ttlInSeconds: 5 * 60,
 });
 
+export async function fetchDatePageProps(
```

Comment:
TODO: extract these functions to separate files _or_ rename file - will do in a separate PR